.github/actions/release/action.yaml

```diff
@@ -140,13 +140,14 @@ inputs:
           os: linux
           arch: x86_64
       ```
-  github-tag-template:
+  github-tag-templates:
     default: '${version}'
     type: string
     description: |
-      GitHub tag template to use for the created release tag. Currently only supported template-var:
+      Comma-separated list of GitHub tag templates to use for the created release tags. Each template can use
```

Comment:
how about using a _newline-separated_ list?

Admittedly, for the "older" actions and workflows in cc-utils, I (unfortunately) followed "docker-build-push"-action's approach of using comma-separated lists (with the downside of resulting in long lines, and bad readability). Later, I learnt that other actions seem to prefer newline-separated inputs, which IMO have a far better readability.